src/abstractions/software/compiler/mockData/index.js

```diff
@@ -0,0 +1,3 @@
+import SyntaxAnalayzerMockData from './syntaxAnalayzer'
```

Comment:
```suggestion
import SyntaxAnalayzerMockData from './SyntaxAnalayzer'
```

Reply:
Is this issue causing the test to fail from your end?

Reply:
And I'm not sure why it's working on my end.

Reply:
Yeah, when I fixed that typo the test run fine on my side

Reply:
Not sure why it didn't fail for you

Reply:
Great, I'll fix it then. But curious to know why it's working with the current way.